docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -720,6 +704,7 @@ Refer to <<data-streams-pipelines>> to learn more.
 * link:{ref}/mapping-source-field.html#synthetic-source[Synthetic `_source`]
 
 These features can be enabled and disabled for {fleet}-managed data streams by using the index template API and a few key settings.
+Note that in versions 8.17.0 and later, Synthetic `_source` requires an Enterprise license.
```

Comment:
I don't know if you want so reuse the same wording as here:
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-source-field.html#:~:text=To%20enable%20this%20subscription%20feature

Reply:
Thanks! I've added that wording, slightly rephrased: `Note that in versions 8.17.0 and later, synthetic `_source` is a subscription feature.`

I've also put the same note in the "Enable synthetic _source` section. The PR description has the latest screencaps.